packages/api/src/routes/medical/patient.ts

```diff
@@ -214,6 +214,30 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /patient/:id/consolidated/query/:requestId
+ *
+ * Returns the status and information on a specific data query for a given patient.
+ *
+ * @param req.param.id The ID of the patient whose data is to be returned.
+ * @param req.param.requestId The ID of the query to be returned.
+ * @returns the status and information on a specific data query for a given patient.
+ */
+router.get(
+  "/consolidated/query/:requestId",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const { patient } = getPatientInfoOrFail(req);
+    const requestId = getFrom("params").orFail("requestId", req);
+    const query = patient.data.consolidatedQueries?.find((q: ConsolidatedQuery) => q.requestId === requestId);
+    if (!query) throw new NotFoundError("Document query not found");
+
+    const respPayload: ConsolidatedQuery = {...query };
+
+    return res.json(respPayload);
```

Comment:
I know the existing `GET /consolidated/query` endpoint is coded like this, but I suggest we specifically indicate the status we're returning, it's clearer.